examples/ds18b20/main.c

```diff
@@ -63,6 +63,13 @@ int main()
                                                         devices[i].id,
                                                         temperature);
 
+            if (temperature_p == NULL) {
+                strncpy(temperature, FSTR("failed to get"),
+                        sizeof(temperature));
+                temperature[sizeof(temperature) - 1] = '\0';
+                temperature_p = temperature;
+            }
+
```

Comment:
Yeah, I agree, it's certainly a good idea to keep an informative message instead of printing ``(null)``.

You'll probably find places in the code base where ``const`` is not properly used. I know there are at least one cast of ``const char *`` to ``char *`` in a file system command callback, which is very bad practice.

In the DS18B120 example one could write:

```
            std_printf(FSTR(),
                       temperature_p != NULL ? temperature_p : "failed to get");
```